### PR TITLE
draft/grw 918/intercom show on scroll

### PR DESCRIPTION
**Pull request by marian-buc**
This adds the ShowOnScrollBehaviour to Intercom to be used whenever we get design team's approval

https://user-images.githubusercontent.com/101566411/160474189-d6eb4d37-88d6-46e1-9158-2b888fa8dd96.mp4

### CODE REVIEW

File: src/client/utils/Intercom.tsx [hunk 1]
```diff
@@ -47,3 +58,78 @@ export const Intercom: React.FC = () => {
...
+      style={{ transition: 'opacity 150ms ease-out', opacity: 1 }}
+    />
+  )
+}
+
+const InjectedStyles = ({ style }: { style: CSSObject }) => (
+  <Global styles={css({ [intercomBtnSelector]: style })} />
+)
+
+const TextLink = styled.a`
```
Comment (simonauner) [inline src/client/utils/Intercom.tsx:115]: Should use object instead of string literal

File: src/client/utils/Intercom.tsx [hunk 2]
```diff
@@ -47,3 +58,78 @@ export const Intercom: React.FC = () => {
...
+  font-size: 0.874rem;
+  color: ${colorsV3.purple900};
+  text-align: center;
+  cursor: pointer;
+`
+const TextLinkVariation = () => {
```
Comment (simonauner) [inline src/client/utils/Intercom.tsx:121]: I'm not sure I understand where this is used - is it shown in the video in the description?

**Comment by dendoesit** [comment]
this should be closed.

**Comment by dendoesit** [comment]
this will be closed for now. we have decided to go with a different solution.

**Pull request closed without merging**